framework/sandstone_test_utils.h

```diff
@@ -55,6 +55,31 @@ class ManagedCodeBuffer {
         free_code_buffer();
     }
 
+    // Default constructor
+    ManagedCodeBuffer() {}
+
+    // Copy constructor
+    ManagedCodeBuffer(const ManagedCodeBuffer& other) : buffer_size(other.buffer_size) {
+        buffer_ptr = static_cast<uint8_t*>(
+            mmap(nullptr, ROUND_UP_TO_PAGE(buffer_size), (PROT_READ | PROT_WRITE), (MAP_ANONYMOUS | MAP_PRIVATE), -1, 0)
+            );
+        std::copy(other.buffer_ptr, other.buffer_ptr + buffer_size, buffer_ptr);
+    }
+
+    // Copy assignment operator
+    ManagedCodeBuffer& operator=(const ManagedCodeBuffer& other) {
```

Comment:
Maybe the correct thing to do here would be to `= delete` these, so no one attempts to copy them.

Reply:
Changed to =delete.
Yes. Coverity still complains, but now we can triage this issue.